charts/csm-replication/values.yaml

```diff
@@ -26,7 +26,6 @@ retryIntervalMin: 1s
 # retryIntervalMax: Maximum retry interval of failed reconcile request
 # Allowed values: time
 retryIntervalMax: 5m
-
```

Comment:
Why delete this emptyline? It seems like its existence is consistent for separation elsewhere.

Reply:
Fixed in https://github.com/dell/helm-charts/pull/473